src/ResourceManager/Websites/ChangeLog.md

```diff
@@ -32,6 +32,10 @@
 * Fixed issue with importing aliases
 * New-AzureRMWebApp - added parameter set for simplified WebApp creation, with local git repository support.
 
+## Version 5.0.0-preview
```

Comment:
Move the content of this into ## Current Release

Reply:
Done